Example/Pods/Pods.xcodeproj/project.pbxproj

```diff
@@ -10,6 +10,7 @@
 		02B2576C95BF77601D378BED30B388AF /* UIViewController+Deselection.swift in Sources */ = {isa = PBXBuildFile; fileRef = CD29029E64EA4205471575F99ED5476E /* UIViewController+Deselection.swift */; };
 		211A3EE904AC58E2EDCAF9211520BEB8 /* Keyboard.swift in Sources */ = {isa = PBXBuildFile; fileRef = F2C31EFD890B03EF4251407B14AC23BB /* Keyboard.swift */; };
 		27E303A10A833D9A0E49F9407276466E /* Pods-Swiftilities_Tests-umbrella.h in Headers */ = {isa = PBXBuildFile; fileRef = A7B341FD9122C117C1415CE785712C68 /* Pods-Swiftilities_Tests-umbrella.h */; settings = {ATTRIBUTES = (Public, ); }; };
+		2B7E4B381D46F93400C21729 /* Integer+Random.swift in Sources */ = {isa = PBXBuildFile; fileRef = 2B7E4B361D46F8AB00C21729 /* Integer+Random.swift */; };
```

Comment:
#req please run `bundle exec pod install` so that these are deterministically generated by CocoaPods.